workflow/applications/gfs_cycled.py

```diff
@@ -119,7 +119,7 @@ def _get_app_configs(self, run):
             configs += ['postsnd']
 
         if options['do_awips']:
-            configs += ['awips']
+            configs += ['awips', 'fbwind']
```

Comment:
Should `workflow/applications/gfs_forecast_only.py` [line 64](https://github.com/DavidHuber-NOAA/global-workflow/blob/28e77758a3fec00b650577952f8e63307b1245fa/workflow/applications/gfs_forecast_only.py#L64) also add `fbwind` when `do_awips` is `True`?

Reply:
Hmm, good question.  I'll give it a test run.